src/components/types.ts

```diff
@@ -136,6 +138,14 @@ export interface TableFormat extends DataFormat {
 
 export type TableData = TableFormat & Updates<TableFormat>
 
+export interface VegaLiteChartFormat extends DataFormat {
```

Comment:
Would it be useful to provide option to configure theme as well ? 
https://github.com/vega/vega-themes

Reply:
Maybe we could use 'default' when light theme and 'dark' with dark theme by default.

Reply:
Should I add light and dark theme to storybook or to the component? If we want people to change theme, maybe add it to storybook makes more sense? Alternatively, we could add additional props for themes so that people could specify their dark and light themes.

Reply:
the current theme can be checked using useTheme. for example,
```
 const isDarkTheme = useTheme().palette.mode === 'dark';
```

Reply:
Do we want to provide a theme props and let people override dark and light theme?

Reply:
yes we could do that and set default values